### DISCUSSION TITLE
Can't download messages from user that have space in nickname by using filter (from:)

### DISCUSSION THREAD

**Discussion by No0negit** [Tyrrrz/DiscordChatExporter#775] [Q&A] [answered]
Hello. First i wanna thank the author for his hard work and amazing program!

Right to my problem. When applying filter "from:" i can't download messages by users that have nicknames which contains 2 words and space between them.
Nickname looks like "Word1 Word2#1111"
I tried these ways:

from:Word1 Word2#1111
from:Word1Word2#1111
from:Word1_Word2#1111
from:Word1-Word2#1111 (gives me an error)
from:Word2#1111
from:Word1 Word2

I'm doing something wrong, or it's just some kind of a bug?

**Comment by 96-LB** [comment, marked as answer]
Use quotes to surround the name. Both `from:"Word 1 Word 2#1111"` and `from:'Word 1 Word 2#1111'` should achieve what you want.

**Reply by No0negit** [reply]
Thank you for help, it's working
